Formula/wxwidgets.rb

```diff
@@ -3,7 +3,7 @@ class Wxwidgets < Formula
   homepage "https://www.wxwidgets.org"
   url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.1/wxWidgets-3.2.1.tar.bz2"
   sha256 "c229976bb413eb88e45cb5dfb68b27890d450149c09b331abd751e7ae0f5fa66"
-  license "wxWindows"
+  license "LGPL-2.0-or-later" => { with: "WxWindows-exception-3.1" }
```

Comment:
Base LGPL2+ is from https://www.wxwidgets.org/about/licence/
```
                     WXWINDOWS LIBRARY LICENCE
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION

This library is free software; you can redistribute it and/or modify it
under the terms of the GNU Library General Public Licence as published by
the Free Software Foundation; either version 2 of the Licence, or (at your
option) any later version.
```

Exception is SPDX replacement for wxWindows.